wagtail/admin/views/scheduled_pages.py

```diff
@@ -0,0 +1,113 @@
+from datetime import timedelta
+
+from django.conf import settings
+from django.core.exceptions import PermissionDenied
+from django.shortcuts import get_object_or_404, redirect
+from django.template.response import TemplateResponse
+from django.utils import timezone
+from django.utils.http import url_has_allowed_host_and_scheme
+from django.utils.translation import gettext_lazy as _
+
+from wagtail.admin import messages
+from wagtail.admin.ui.components import Component
+from wagtail.models import Page
+from wagtail.permission_policies.pages import PagePermissionPolicy
+
+
+def get_scheduled_pages_for_user(request):
+    user_pages = PagePermissionPolicy().instances_user_has_permission_for(
+        request.user, "publish"
+    )
+    pages = (
+        Page.objects.annotate_approved_schedule()
+        .filter(_approved_schedule=True)
+        .prefetch_related("content_type")
+        .order_by("-first_published_at")
+        & user_pages
+    )
+
+    if getattr(settings, "WAGTAIL_I18N_ENABLED", False):
+        pages = pages.select_related("locale")
+    return pages
+
+
+class ScheduledPagesPanel(Component):
+    name = "scheduled_pages"
+    template_name = "wagtailadmin/scheduled_pages/panel.html"
+    order = 200
+
+    def get_context_data(self, parent_context):
+        request = parent_context["request"]
+        context = super().get_context_data(parent_context)
+        context["pages_to_be_scheduled"] = get_scheduled_pages_for_user(request)
+        context["request"] = request
+        context["csrf_token"] = parent_context["csrf_token"]
+        return context
+
+
+def publish(request, page_id):
+    page = get_object_or_404(Page, id=page_id).specific
+    if not page.permissions_for_user(request.user).can_publish():
+        raise PermissionDenied
+
+    new_go_live_timestamp = timezone.now() - timedelta(seconds=1)
```

Comment:
Question: Why is it going live in the past?

Reply:
This use case is when the editor or admin wants to override the schedule, and publish the scheduled changes immediately.